wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+- **Long-term slider velocity**: Un cambio de velocidad del slider que dura más de 4 compases.
+- **Musical phrase**: Una unidad independiente de metro musical que suele tener 4 compases de largo.
+- **Short-term slider velocity**: Un cambio en la velocidad del slider que dura menos que 1/2 de un pulso.
+
+### Patrones
+
+- **Anchor**: Un patrón que tiene una cantidad relativamente alta de notas en una columna comparada con las otras, usualmente se encuentran entre diferentes tipos de streams.
+- **Bracket**: Un patrón que involucra múltiples trills usando columnas cerca una de las otras.
+- **Chord**: Dos o más notas presionadas al mismo tiempo.
+- **Chordstream (Jumpstream, Handstream, Quadstream)**: Streams que mezclan diferentes tipos de chords.
+- **Grace**: Dos o más notas presionadas en sucesión rápida, generalmente usando snaps de 1/6 o más.
+- **Hand**: Tres notas presionadas al mismo tiempo. Ocasionalmente referida como una "triple".
+- **Jack**: Tres o más notas consecutivas en la misma columna, usualmente presionada a un ritmo más rápido que las notas aledañas.
+- **Jump**: Dos notas presionadas al mismo tiempo. Ocasionalmente referida como una "doble".
+- **Jumptrill**: Un trill consistiendo en dos jumps que alternan.
+- **Minijack**: Dos notas consecutivas en la misma columna, usualmente presionadas a un ritmo más rápido que las notas aledañas.
+- **Release**: El final de una nota larga el cual debe soltarse en una cierta ventana de tiempo para poder conseguir el puntaje completo apropiadamente.
+- **Roll**: Una variación del stream que consiste de notas que van desde la primera columna hasta lo ultima o desde la última hasta la primera, usualmente en sucesión rápida.
+- **Shield**: Una nota que aparece en la misma columna inmediatamente antes del inicio de una nota larga.
+- **Split-jumptrill**: Un jumptrill que requiere el uso de las dos manos al mismo tiempo para ser realizado correctamente (por ejemplo, dos trills de una mano).
+- **Stream**: Notas constantes con el mismo intervalo por al menos una pulsación que no se repite para crear un minijack.
+- **Trill**: Más de dos notas que alternan entre dos columnas sin interrupción.
+
+Consulte [Guía de mapping de osu!mania](/wiki/Guides/osu!mania_Mapping_Guide) para obtener más terminología de patrones.
+
+## General
+
+Las reglas y pautas generales se aplican a todo tipo de dificultad de osu!mania. Las reglas y pautas relacionadas con el ritmo se aplican a beatmaps de aproximadamente 180 BPM con compases de 4/4. Si tu canción es drásticamente más rápida o más lenta, algunas variables pueden ser diferentes, como se detalla en [Escalado de BPM en los criterios de clasificación](/wiki/Ranking_Criteria/Scaling_BPM)
+
+### Reglas
+
+- **El mod `Auto` debe ser capaz de alcanzar la puntuación completa (1,000,000) en todas las dificultades.**

```

Comment:
```suggestion
- **El mod `Auto` debe ser capaz de alcanzar la puntuación completa (1&nbsp;000&nbsp;000) en todas las dificultades.**
```
According to _Ortografía de la lengua española_ ([1](http://aplica.rae.es/orweb/cgi-bin/v.cgi?i=UeJUPXgbzfJDiskW)/[2](http://aplica.rae.es/orweb/cgi-bin/v.cgi?i=KGJKfgxvLmYwaJqK)), _[Diccionario panhispánico de dudas](https://www.rae.es/dpd/números)_  and [Resolution 10 of the 22nd CGPM (2003)](https://www.bipm.org/en/committees/cg/cgpm/22-2003/resolution-10), "neither dots nor commas are ever inserted in the spaces between groups".
![Warning from page 664 of Ortografía de la lengua española](https://user-images.githubusercontent.com/58957468/204353056-d2e9b2af-7ad9-4ee4-9279-c55543337a39.png)